src/internal/Subscription.ts

```diff
@@ -66,6 +66,7 @@ export class Subscription implements SubscriptionLike {
       }
 
       const { initialTeardown } = this;
+      this.initialTeardown = undefined;
```

Comment:
I'd use `null!` here, I think (not sold on it), just because `null` tells me that _we_ set it, rather than it just never being set.